test_unstructured_ingest/python/test-ingest-astra-output.py

```diff
@@ -0,0 +1,56 @@
+import click
+
+from astrapy.db import AstraDB
+
+@click.command()
+@click.option("--token", type=str)
+@click.option("--api-endpoint", type=str)
+@click.option("--collection-name", type=str, default="collection_test")
+def run_check(token, api_endpoint, collection_name):
+    print(f"Checking contents of Astra DB collection: {collection_name}")
+
+    # Initialize our vector db
+    astra_db = AstraDB(token=token, api_endpoint=api_endpoint)
+    astra_db.delete_collection(collection_name)
+    astra_db_collection = astra_db.create_collection(collection_name, dimension=5)
+
+    # Insert a document into the test collection
```

Comment:
This seems wrong. We should not have to insert a document. or create_collection. We already did that in the shell script that ran before this python file is called.

All we should be doing in this file 
1) do a vector search based on the embedding we uploaded (book-war-and-peace-1p.txt)
2) doing a count
3) deleting the collection

Reply:
Here is one way we've been doing the vector search:
https://github.com/Unstructured-IO/unstructured/blob/main/scripts/elasticsearch-test-helpers/destination_connector/test-ingest-elasticsearch-output.py
But you can choose your own way.